### PR TITLE
Bump pgxn to version 10.2.4

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: pkgvars [hunk 1]
```diff
@@ -1,4 +1,4 @@
 pkgname=citus
 pkgdesc='Citus (Open-Source)'
-pkglatest=10.2.3
+pkglatest=10.2.4
 releasepg=12,13,14
```
Comment (hanefi) [inline pkgvars]: `10.2` supports PG 11, 12, and 13.

14 is not supported.

Can we remove these references in pkgvars as they are not used?

Comment (gurkanindibay) [reply pkgvars]: No we can not we don't use postgres matrix here

Comment (hanefi) [reply pkgvars]: My comment above is not correct.

10.2 supports 12, 13 and 14.

10.1 supports 11, 12, and 13.

My bad.

However, I still think that we do not need the values here.

Comment (gurkanindibay) [reply pkgvars]: I removed these parameters

**Review by hanefi — APPROVED** [review]

**Pull request merged**